eva-accession-release-automation/run_release_in_embassy/create_release_tracking_table.py

```diff
@@ -152,23 +152,25 @@ def fill_should_be_released_for_taxonomy(self, tax, asm_sources):
             self.fill_should_be_released_for_taxonomy_and_assembly(tax, asm_acc, asm_sources[asm_acc])
 
     def fill_should_be_released_for_taxonomy_and_assembly(self, tax, asm, src):
-        """Fills should_be_released for a taxonomy/assembly pair based on whether there are current RS IDs,
-        and updates the sources column as well.
-        TODO Also check for deprecated (https://www.ebi.ac.uk/panda/jira/browse/EVA-3402)"""
+        """Fills should_be_released for a taxonomy/assembly pair based on whether there are current RS IDs or
+        deprecated RS IDs and updates the sources column as well."""
         should_be_released_eva = should_be_released_dbsnp = False
         if asm != 'Unmapped':
             ss_query = {'tax': tax, 'seq': asm, 'rs': {'$exists': True}}
             rs_query = {'tax': tax, 'asm': asm}
+            rs_opt_query = {'inactiveObjects.tax': tax, 'inactiveObjects.asm': asm, 'eventType': 'DEPRECATED'}
```

Comment:
I don't think this will have any effect but the queries on `tax` should be done on SubmittedVariantEntity and SubmittedVariantOperationEntity. Not on their ClusteredVariants equivalent.